distributions/nuxt-start/package.js

```diff
@@ -1,23 +1,23 @@
 export default {
   build: true,
-  extend(pkg, { load }) {
-    pkg.on('build:done', () => {
-      const mono = load('../..')
-      const nuxt = load('../nuxt')
+  hooks: {
+    async 'build:done'(pkg) {
+      const mono = pkg.load('../..')
+      const nuxt = pkg.load('../nuxt')
 
-      pkg.copyFilesFrom(mono, [
+      await pkg.copyFilesFrom(mono, [
```

Comment:
Could we probably `Promise.all` this + the next copyFieldsFrom?

Reply:
I didn't make it for pkg code readability. Sometimes clear code worth over little (Maybe matter of ms) perf improvements.  :D

Reply:
Yeah, especially for the Nuxt build process that will only be issued by maintainers or CI ☺️